tools/test/h5dump/CMakeTests.cmake

```diff
@@ -1172,6 +1174,10 @@
   # test to verify HDFFV-9407: long double full precision
   ADD_H5_GREP_TEST (t128bit_float 1 "1.123456789012345" -m %.35Lf t128bit_float.h5)
 
+  # test to verify HDFFV-10480: out of bounds read in H5O_fill_new[old]_decode
+  ADD_H5_TEST (tCVE_2018_11206_fill_old 1 tCVE_2018_11206_fill_old.h5)
```

Comment:
I wonder if ADD_H5_TEST should be ADD_H5_GREP_TEST and search that the test failure that is expected and doesn't fail for some other reason.

Reply:
I'm not sure, but TOOLTEST_FAIL took care of that, it would detect the segfault.  I wish the author would put some comments to these macros... :-/

Reply:
Oh it segfaults - okay then this is fine.